### ISSUE TITLE
Ship our setup.cfg

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#832]
🐞 **Problem**

Our pytest configuration is not being picked up by users that want to run the test suite because we are not shipping our `setup.cfg`.

Also, other packages mistakenly shipping `setup.cfg` to the wrong location can interfere in poliastro tests, see #750 and #784.

💡 **Possible solutions**

Ship our own `setup.cfg`. Using setuptools, this was possible doing

```
package_data={
    "poliastro": "setup.cfg"
}
```

(see [this statsmodels discussion](https://github.com/statsmodels/statsmodels/issues/6114))

Not sure if flit supports this.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by priyanshurohilla** [comment]
I want to work on this.

**Comment by susanka068** [comment]
I'd like to work on this issue

**Comment by Juanlu001** [comment]
@susanka068, @priyanshurohilla is already working on it at #836

**Comment by noc0lour** [comment]
This can be closed I assume.

**Comment by Juanlu001** [comment]
Thanks for the reminder @noc0lour. I wanted to test this before closing... and I saw we're not there yet.

We added `setup.cfg` under `sdist`, but it's not being included in the wheel. I'll see what can be done.

**Comment by Juanlu001** [comment]
Well, I don't think it can be easily done for a file that is out of the `src` tree. This is probably a symptom that we shouldn't be shipping our tests, even though other packages in the scientific Python ecosystem do it.

Out of curiosity, I checked SciPy wheels and, even though they have `include tox.ini pytest.ini` in their `MANIFEST.in`, their pytest configuration doesn't make it to the wheels.

It's OK that we managed to include `setup.cfg` in our source distribution, but perhaps we should think about removing the tests from the wheels. I won't even bother about opening an issue for it.

In the meanwhile, I close this.
